drivers/at86rf2xx/include/at86rf2xx_internal.h

```diff
@@ -34,8 +34,14 @@ extern "C" {
 /**
  * @brief   Transition time from SLEEP to TRX_OFF in us, refer figure 7-4, p.42.
  *          For different environments refer figure 13-13, p.201
+ *          For AT86RF212B, delay is longer, see page 42, tTR1a: SLEEP⇒CLKM is available
+ *          http://ww1.microchip.com/downloads/en/DeviceDoc/Atmel-42002-MCU_Wireless-AT86RF212B_Datasheet.pdf
  */
+#ifdef MODULE_AT86RF212B
+#define AT86RF2XX_WAKEUP_DELAY          (1100U)
```

Comment:
I think 1000 should be enough, you'll get a margin anyway because of processing delays before and after the 1000 us wait.